conf/etc/cron.hourly/cortx_ha_log.conf

```diff
@@ -2,9 +2,9 @@
 /var/log/seagate/cortx/ha/*.log
 {
     rotate 10
-    size 10M
+    size 10K
```

Comment:
Why this size change @VijayThakkar132 ?

Reply:
Actually, cron.hourly get executed per hour and check the log file size if the size is more than the specified size then only perform the rotate operation. I thought 10K is sufficient.

Reply:
no, I think we need to restore this back to 10M

Reply:
Sure. Will do that

Reply:
Done